test/test_utils/test_utils.jl

```diff
@@ -0,0 +1,311 @@
+#
+# Load dependencies
+#
+
+using AbstractMCMC
+using AdvancedMH
+using Clustering
+using Distributions
+using Distributions.FillArrays
+using DistributionsAD
+using FiniteDifferences
+using ForwardDiff
+using MCMCChains
+using NamedArrays
+using Optim
+using Optimization
+using OptimizationOptimJL
+using PDMats
+using ReverseDiff
+using SpecialFunctions
+using StatsBase
+using StatsFuns
+using Tracker
+using Turing
+using Turing.Inference
+using Turing.RandomMeasures
+using Zygote
+
+using LinearAlgebra
+using Pkg
+using Random
+using Test
+using StableRNGs
+
+using AdvancedPS: ResampleWithESSThreshold, resample_systematic, resample_multinomial
+using AdvancedVI: TruncatedADAGrad, DecayedADAGrad, apply!
+using Distributions: Binomial, logpdf
+using DynamicPPL: getval, getlogp
+using ForwardDiff: Dual
+using MCMCChains: Chains
+using StatsFuns: binomlogpdf, logistic, logsumexp
+using TimerOutputs: TimerOutputs, @timeit
+using Turing: BinomialLogit, ForwardDiffAD, Sampler, SampleFromPrior, NUTS, TrackerAD,
+                Variational, ZygoteAD, getspace
+using Turing.Essential: TuringDenseMvNormal, TuringDiagMvNormal
+import LogDensityProblems
+import LogDensityProblemsAD
+
+
+#
```

Comment:
Do we feel this is more readable than what we did before?

As in, why don't we just keep the separate files but include all of them in a single file?